firestore/query.go

```diff
@@ -364,6 +366,97 @@ func (q Query) Deserialize(bytes []byte) (Query, error) {
 	return q.fromProto(&runQueryRequest)
 }
 
+// DistanceMeasure is the distance measure to use when comparing vectors.
+type DistanceMeasure int32
+
+const (
+	// Measures the EUCLIDEAN distance between the vectors. See
+	// [Euclidean](https://en.wikipedia.org/wiki/Euclidean_distance) to learn
+	// more
+	DistanceMeasureEuclidean DistanceMeasure = 1
```

Comment:
`DistanceMeasure(pb.StructuredQuery_FindNearest_EUCLIDEAN)` instead of 1.
This way we guarantee that they are the same.

Reply:
Modified